Samples/outlook-encrypt-attachments/src/commands/commands.js

```diff
@@ -0,0 +1,559 @@
+var totalDistributionLists = 0;
+// eslint-disable-next-line no-unused-vars
+Office.initialize = function (reason) {
+  try {
+    console.log(`Office.initialize(): Huzzah!`);
+  }
+  catch(ex){
+    console.error(`Office.initialize(): Error! ${ex}`);
+  }  
+};
+/**
+ * Method that fires when a appointment is being created or edited
+ * @param {Office.AsyncResult} event default: Office.AsyncResult
+ */
+function onAppointmentComposeHandler(event) {
+  //NOTE: Must call event.completed() when any logic below is finished!
+
+  console.log("onAppointmentComposeHandler(): entered!");
+
+  let originalAppointmentDate = {}; //Create an object to cache the original date/time and persist it to localStorage
+
+  Office.context.mailbox.item.start.getAsync((asyncResult) => {
+    if (asyncResult.status !== Office.AsyncResultStatus.Succeeded) {
+      console.error(`onAppointmentComposeHandler(): Action failed with message ${asyncResult.error.message}`);
+      event.completed();
+      return;
+    }
+    console.log(`onAppointmentComposeHandler(): Appointment starts: ${asyncResult.value}`);
+    originalAppointmentDate.start = asyncResult.value;
+
+    Office.context.mailbox.item.end.getAsync((asyncResult2) => {
+      if (asyncResult2.status !== Office.AsyncResultStatus.Succeeded) {
+        console.error(`onAppointmentComposeHandler(): Action failed with message ${asyncResult2.error.message}`);
+        event.completed();
+        return;
+      }
+      originalAppointmentDate.end = asyncResult2.value;
+
+      console.log(`onAppointmentComposeHandler(): Appointment ends: ${asyncResult2.value}`);
+      console.log("onAppointmentComposeHandler(): Setting SessionData...");     
+
+      Office.context.mailbox.item.sessionData.setAsync("appointment_info", JSON.stringify(originalAppointmentDate), function(asyncResult3){
+        if (asyncResult3.status === Office.AsyncResultStatus.Succeeded) {
+          console.log("onAppointmentComposeHandler(): sessionData set");
+          //Add a notification message to ask the user to open the Task Pane to view additional information on the sample
+          //NOTE: Clicking the "Show Task Pane" link in the InfoBar doesn't work in Outlook Online. A fix is in progress and being tested: https://github.com/OfficeDev/office-js/issues/2125
+                      
+          console.log("onAppointmentComposeHandler(): Adding notification message...");
+
+          //NOTE: actions array only applicable to insightMessage types
+          Office.context.mailbox.item.notificationMessages.addAsync(
+            "showInfoBarForSampleInstructions",
+            {
+              type: Office.MailboxEnums.ItemNotificationMessageType.InsightMessage,
+              message:
+                "Open the Task Pane for details about running the Outlook Event-based Activation Sample Add-in",
+              icon: "Icon.16x16",
+              actions: [
+                {
+                  actionText: "Show Task Pane",
```

Comment:
```suggestion
                  actionText: "Show task pane",
```

Reply:
Eric can you please search and replace to "Show task pane" (task pane is lowercase.) thanks!